src/apps/opening-hours-editor/OpeningHoursEditor.tsx

```diff
@@ -57,9 +62,9 @@ const OpeningHoursEditor: React.FC<OpeningHoursEditorType> = ({
         ref={fullCalendarRef}
         plugins={[dayGridPlugin, timeGridPlugin, interactionPlugin]}
         headerToolbar={{
-          left: "title",
-          center: "prev,next today",
-          right: "dayGridMonth,timeGridWeek"
+          left: "dayGridMonth,timeGridWeek",
+          center: "title",
+          right: "prev,next today"
```

Comment:
I wonder why you changed this.

I can see that you put in a separate commit but I do not understand the reasoning behind the change. It is not mentioned in the PR or in the ticket.

Reply:
I found it annoying that the button changes position when I click through the months.